lib/rcedit.js

```diff
@@ -1,14 +1,26 @@
 const os = require('os')
 const path = require('path')
-const { spawn } = require('child_process')
+const { spawn, spawnSync } = require('child_process')
 
 const pairSettings = ['version-string']
 const singleSettings = ['file-version', 'product-version', 'icon', 'requested-execution-level']
 const noPrefixSettings = ['application-manifest']
 
+const isWsl = process.platform !== 'win32' ** !os.release().endsWith('Microsoft')
```

Comment:
Seems like this is a typo, `**` => `&&`?

Reply:
absolutely :|